.github/workflows/on-merge.yml

```diff
@@ -110,3 +92,21 @@ jobs:
       - name: Dispatch event to docs repo
         run: |
           pulumictl dispatch -c pulumi-cli-dev-version -r pulumi/docs dev_version=${{ steps.ghd.outputs.describe }}
+
+  ci-ok:
+    name: ci-ok
+    needs: [ci, prepare-release, dev-release, update-dev-version]
+    if: always() # always report a status
+    runs-on: ubuntu-latest
+    steps:
+      - name: CI failed
+        if: ${{ needs.ci.result != 'success' }}
+        run: exit 1
+      - name: Release failed
+        if: ${{ needs.prepare-release.result != 'success' }}
+        run: exit 1
+      - name: Dev release failed
```

Comment:
Should this check that update dev version worked?

Reply:
Probably a good idea, yes.  Done.